verification/curator-service/api/src/controllers/auth.ts

```diff
@@ -6,14 +6,18 @@ import {
 import { NextFunction, Request, Response } from 'express';
 import { User, UserDocument } from '../model/user';
 
-import { Strategy as BearerStrategy } from 'passport-http-bearer';
+import {
+    Strategy as BearerStrategy,
+    IVerifyOptions,
+} from 'passport-http-bearer';
 import { Router } from 'express';
 import axios from 'axios';
 import passport from 'passport';
 
-// mustBeAuthenticated is a middleware that checks that the user making the call is
-// authenticated.
-// Subsequent request handlers can be assured req.user will be defined.
+/**
+ * mustBeAuthenticated is a middleware that checks that the user making the call is authenticated.
+ * Subsequent request handlers can be assured req.user will be defined.
```

Comment:
Should subsequent text beyond the one-line summary be `@remarks`?

Reply:
I don't understand when `@remarks` is supposed to be used... is this documented anywhere?
In doubt I decided not to use it at all ;-) to me everything is a remark somehow...